app/src/main/java/org/oppia/app/player/state/hintsandsolution/SolutionViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.player.state.hintsandsolution
+
+import androidx.lifecycle.ViewModel
+
+/** [ViewModel] for title in [HintsAndSolutionFragment]. */
+class SolutionViewModel : HintsAndSolutionItemViewModel(){
+  var solutionSummary: String = ""
+  var correctAnswer: String = ""
+  var title: String = ""
+  var isSolutionRevealed: Boolean = false
+  var solutionCanBeRevealed: Boolean = false
```

Comment:
Please make these all `var` as oberservale field or pass them to constructor as val

Reply:
+1

Reply:
Done.